src/classes/Cart.ts

```diff
@@ -231,7 +231,11 @@ abstract class Cart {
 
         this.offer.data('handleTimestamp', moment().valueOf());
 
-        this.offer.setMessage('Powered by TF2 Automatic');
+        if (!process.env.OFFER_MESSAGE === undefined || !process.env.OFFER_MESSAGE) {
```

Comment:
Just change this to !process.env.OFFER_MESSAGE

Reply:
Oh.. for both cases? I see. Ok, nick.